### DISCUSSION TITLE
magnetic tape is disappear

### DISCUSSION THREAD

**Discussion by Yoo-Youngjae** [HumanSupportRobot/QandA#49] [General]
Hi.

While we are preparing for the robocup,
we confirmed that the magnetic tape was gone.
I experienced last year that it is essential for the Robocup arena to work.

Can we get a magnetic tape?

Thank you.
![KakaoTalk_20230513_162815506](https://github.com/HumanSupportRobot/QandA/assets/44061214/758518c1-0870-488c-aa8e-3f4877d35fad)

**Comment by hsr-admin** [comment]
An applicable magnetic tape could be Accurate Systems, Inc. GT-100 N-type magnetic tape.

Is it possible to purchase those magnetic tapes?
(Reference.)
https://docs.hsr.io/hsr_customization_manual_en/modification.html#disabling-the-fall-prevention-switches

**Reply by Yoo-Youngjae** [reply]
I searched, but couldn't find a link to buy it.
Can you recommend a link that can ship to the South Korea?

Thank you.
